src/internal/reflection.js

```diff
@@ -39,7 +39,7 @@ const typeofString = "string";
  * @returns {boolean} `true` if property is an own-property, `false` otherwise.
  */
 export function hasOwn(object, property) {
-  return Object.prototype.hasOwnProperty.call(object, property);
+  return Object.hasOwn(object, property);
```

Comment:
This change - which is due to [`prefer-object-has-own`](https://eslint.org/docs/latest/rules/prefer-object-has-own) - is not compatible with Node.js v14.18.0, see:

https://github.com/ericcornelissen/shescape/actions/runs/12340622366/job/34438335681?pr=1815#step:8:22

Reply:
Resolved with 9d8251be9f3377bf1eb398c832a96bae171c572d, still need to open an issue to enable the rule and refactor this code after this has been merged.

Reply:
Fully resolved with https://github.com/ericcornelissen/shescape/issues/1818